android/src/main/java/com/gr4vy/embedreactnative/EmbedReactNativeModule.java

```diff
@@ -61,6 +66,25 @@ public static <T> T coalesce(T... items) {
     return null;
   }
 
+  public static Object getMapOptionalValue(ReadableMap map, String key) {
```

Comment:
`ReadableMap` doesn't seem to have a method to get options values and fallback to `null`, so adding one here